AGPs/AGP-Authorship-Rewards-Program.md

```diff
@@ -0,0 +1,42 @@
+---
+AGP: N/A
+Title: AGP Authorship Rewards Program
+Author: burrrata (@burrrata)
+Status: Stage IV
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: AGP Authorship Rewards Program
+
+## Description of desired Association policy change
+
+The Aragon Association will reward AGP authors and contributors with ANT at the end of every ANV cycle after ANT votes have been finalized and announced. The ANT rewards will be:
+- 300 ANT to be split between any listed authors of an AGP that is approved by the AA for an upcoming ANV.
+- 200 ANT to be split between any listed contributors of an AGP that is approved by the AA for an upcoming ANV.
+- 300 ANT to be split between any listed authors of an AGP that is approved by ANT voters.
+- 200 ANT to be split between any listed contributors of an AGP that is approved by ANT voters.
+
+To receive authorship rewards AGP authors must include an Ethereum address which can receive the ANT ERC-20 token in the proposal. This can be added to the `Author` section in the header of the AGP. Names and addresses listed in the `Author` section of the header should be treated as authors or co-authors in regards to AGP authorship rewards.
```

Comment:
```suggestion
To receive authorship rewards AGP authors must include an Ethereum address that can receive the ANT ERC-20 token in the proposal. This can be added to the `Author` section in the header of the AGP. Names and addresses listed in the `Author` section of the header should be treated as authors or co-authors in regards to AGP authorship rewards.
```